spec/controllers/admin/emergency_message_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Admin::EmergencyMessagesController do
+  before :each do
+    @user = User.create!(name: 'hello', admin: true, email: 'lol@biz.info', password: 'irrelevant')
```

Comment:
That would work but for consistency it'd have to be changed everywhere which is outside the scope of these sprints.